Jenkinsfile.integration

```diff
@@ -37,11 +39,17 @@ pipeline {
         }
 
         stage('Setup network') {
+            options {
+                timeout(time: 45, unit: 'MINUTES', activity: true)
```

Comment:
This should be matter of few minutes tops

Reply:
Agreed, I was just copying in what the default was. Note that it's also based on activity, so it's 45min of no stdout (afaict, as it's not well documented). So copying the same activity timeout to each section isn't giving it any more than what it previously had.

That said, we can find tune them in this PR if you think it's necessary?